frameworks/Java/officefloor/source_files

```diff
@@ -0,0 +1,4 @@
+src/woof_benchmark/src/main/java/net/officefloor/benchmark/Fortune.java
+src/woof_benchmark/src/main/java/net/officefloor/benchmark/FortunesLogic.java
+src/woof_benchmark/src/main/java/net/officefloor/benchmark/Logic.java
+src/woof_benchmark/src/main/java/net/officefloor/benchmark/World.java
```

Comment:
Remove this `source_files` file, please.  We used to use files like this to help us count the lines of code for each framework, but we don't do that any longer.  We fixed our README to not ask people to create these files anymore: https://github.com/TechEmpower/FrameworkBenchmarks/commit/865a28ba19c2e36374ed664aeedef893df2fce9b